style.css

```diff
@@ -0,0 +1,300 @@
+html {
+    box-sizing: border-box;
+    font-size: 62.5%;
+    scroll-behavior: smooth;
+  }
+  
+  .fullpage {
+    padding-bottom: 0;
+    margin: 0;
+    display: -webkit-box;
+    display: -webkit-flex;
+    display: -ms-flexbox;
+    display: flex;
+    -webkit-box-orient: vertical;
+    -webkit-box-direction: normal;
+    /* -webkit-flex-direction: column; */
+    -ms-flex-direction: column;
+    flex-direction: column;
+  }
+  
+  .nav .list-item {
+    display: inline;
+    padding: 12px;
+    color: white;
+    margin-left: 200px;
+    font-weight: bolder;
+  }
+  
+  .nav .list-item a {
+    color: white;
+  }
+  
+  .fullpage .main-header {
+    min-width: 0;
+  }
+  
+  .projects-section {
+    text-align: center;
+    padding: 10rem 2rem;
+    background: var(--main-blue);
+  }
+  
+  .welcome-section {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    width: 100%;
+    height: 100vh;
+    background-color: #000;
+    background-image: linear-gradient(62deg, #3a3d40 0%, #181719 100%);
+  }
```

Comment:
Good job so far. You may need to add a contrasting colour to the welcome title to make it pop

Reply:
- [ ] I think it would be better to add a contrasting color to your welcome title as to give it nice UI look.